src/Car.java

```diff
@@ -0,0 +1,10 @@
+public class Car extends Transport {
+
+    public Car(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
+    }
+
+    public void checkEngine() {
+        System.out.println("Проверяем двигатель");
```

Comment:
```suggestion
        System.out.println("Проверяем двигатель машины");
```
реализация метода должна показывать полиморфизм - проверить двигатель у машины и у грузовика - это разные вещи, а у тебя одно и то же